spring-cloud-gcp-samples/spring-cloud-gcp-integration-pubsub-sample/README.adoc

```diff
@@ -1,13 +1,26 @@
 = Spring Integration Channel Adapters for Google Cloud Pub/Sub Code Sample
 
 This code sample contains a couple of apps.
-One app gets a message provided by the user through a webpage and publishes it to a Google Cloud Pub/Sub topic.
-The other app subscribes to that topic and logs received messages.
 
+* Pub/Sub Sender & Receiver Applications:
++
+The `sample-sender` and `sample-receiver` applications demonstrate how to send messages from one
+application to another using Pub/Sub. The `sample-sender` provides a web interface to send messages
+to a topic whose messages are received by the `sample-receiver`.
++
 The same content is also available in the
 https://codelabs.developers.google.com/codelabs/cloud-spring-cloud-gcp-pubsub-integration/index.html[Messaging with Spring Integration and Google Cloud Pub/Sub] codelab.
 
-== Setup & Configuration
+* JSON Payload Application:
++
+The `sample-json-payloads` application demonstrates how the Cloud GCP Pub/Sub APIs can
+automatically serialize and deserialize JSON payloads that are published to topics or received
+by subscribers using Jackson converters.
+
+== Base Setup & Configuration
+
+Before running the applications, you must first configure your GCP project and set up the Pub/Sub
+topic and subscription.
 
 1. Configure your GCP project ID and credentials by following link:../../spring-cloud-gcp-docs/src/main/asciidoc/core.adoc#project-id[these instructions].
```

Comment:
I learned recently that you can just do ". Configure..." without specifying the number, and it works too.